Userland/Applets/ClipboardHistory/main.cpp

```diff
@@ -17,14 +18,20 @@
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    TRY(Core::System::pledge("stdio recvfd sendfd rpath unix"));
+    TRY(Core::System::pledge("stdio recvfd sendfd rpath unix cpath wpath"));
     auto app = TRY(GUI::Application::try_create(arguments));
+    auto clipboard_config = TRY(Core::ConfigFile::open_for_app("KeyboardSettings"));
+    bool persistent_clipboard = clipboard_config->read_bool_entry("StartupEnable", "PersistentClipboard", false);
+
+    StringBuilder clipboard_file_path_builder;
+    clipboard_file_path_builder.append(Core::StandardPaths::home_directory());
+    clipboard_file_path_builder.append("/.clipboard"sv);
```

Comment:
Here is a limitation of putting this in `~/.config`:

The user can now configure the path to the persistent clipboard thanks to the following UI:
![image](https://user-images.githubusercontent.com/8809698/185813282-ddba401c-6147-4ec7-b3f9-1567999db901.png)

But if this value is changed, the `ClipboardHistory` applet is not restarted, and there is a **very** low chance that the current instance has unveiled the new path. So, as I implemented it so far, the applet needs to be restarted after every change of this value.

Nonetheless, I find it very handy to be able to configure it.

Reply:
I'm not sure how that's a problem? Given the new information, it could re-exec itself, or just not unveil anything; keeping the configurable history path is fine, I'm just saying it should live in ~/.config by default